app/controllers/course/forum/forums_controller.rb

```diff
@@ -74,4 +76,22 @@ def load_forum
   def forum_params
     params.require(:forum).permit(:name, :description, :course_id)
   end
+
+  # @return [Course::ForumsComponent | nil] The forum component or nil if disabled.
+  def component
+    current_component_host[:course_forums_component]
+  end
+
+  # Ensure that the component is enabled.
+  #
+  # @raise [Coursemology::ComponentNotFoundError] When the component is disabled.
+  def check_component
+    fail ComponentNotFoundError unless component
+  end
+
+  def add_forum_breadcrumb
+    add_breadcrumb component.settings.title || :index, course_forums_path(current_course)
+    add_breadcrumb @forum.name, course_forum_path(current_course, @forum) unless @forum.nil? ||
+      @forum.new_record?
```

Comment:
Align the operands of a condition in an `unless` statement spanning multiple lines.